pkg/daemon/discover/discover_test.go

```diff
@@ -100,10 +100,13 @@ testa2   centos_host13-home
 
 	context := &clusterd.Context{Executor: executor}
 
-	devices, err := probeDevices(context)
-	assert.Nil(t, err)
-	assert.Equal(t, 1, len(devices))
-	assert.Equal(t, "ext2", devices[0].Filesystem)
+	useCVInventory := true
+	if useCVInventory {
```

Comment:
The unit test could always run this, right? No need for a var that is always true.

Reply:
right! :-) thx!